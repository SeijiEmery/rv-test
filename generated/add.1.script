load /x 0 generated/add.1.hex
writereg 1 10
writereg 2 10
run 0 1000
readreg 1
readreg 2