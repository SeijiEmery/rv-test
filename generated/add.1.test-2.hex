0000000    93  00  a0  00  13  81  a0  00  73  00  10  00                
0000014
