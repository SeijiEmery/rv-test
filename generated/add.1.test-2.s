addi x1, x0, 10
addi x2, x1, 10# Note that test names are optional (and currently unused)

nop
nop
nop
nop
nop
ebreak
