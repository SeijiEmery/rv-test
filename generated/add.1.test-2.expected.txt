tests/add.test.s:10
R1 = 10
R2 = 20
